forge-gui/res/cardsfolder/rebalanced/a-civil_servant.txt

```diff
@@ -0,0 +1,9 @@
+Name:A-Civil Servant
+ManaCost:G W
+Types:Creature Cat Citizen
+PT:2/3
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME attacks, you may tap another untapped Citizen you control. If you do, Civil Servant gets +1/+0 and gains trample until end of turn.
+SVar:TrigPump:AB$ Pump | Cost$ tapXType<1/Creature.Citizen/Citizen> | Defined$ Self | NumAtt$ 1 | KW$ Trample
```

Comment:
does this needs to be `another` or is that build in?

Reply:
right, needs to be <1/Citizen.Other> (also non A- version) – THEN the cost parsing will do it properly

Reply:
yeah, you should be able to tap non creatures, (Tribal Enchantment)